src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/jpa/entity/AssessmentSkipEntity.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.jpa.entity
+
+import jakarta.persistence.Entity
+import jakarta.persistence.EntityListeners
+import jakarta.persistence.EnumType
+import jakarta.persistence.Enumerated
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import org.springframework.data.annotation.CreatedBy
+import org.springframework.data.annotation.CreatedDate
+import org.springframework.data.jpa.domain.support.AuditingEntityListener
+import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data.AssessmentSkipReason
+import java.time.LocalDateTime
+
+@Entity
+@Table(name = "assessment_skip")
+@EntityListeners(AuditingEntityListener::class)
+data class AssessmentSkipEntity(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long? = null,
+  val assessmentType: ResettlementAssessmentType,
+  val prisonerId: Long,
+  @Enumerated(EnumType.STRING)
+  val reason: AssessmentSkipReason,
+  val moreInfo: String? = null,
+  @CreatedBy
```

Comment:
if this is the only auditable class ok, but otherwise you could extend another Jpa Entity with auditing fields createdBy and creationDate over there

Reply:
There are other classes with similar fields but we're manually passing through the headers from the controller. I think I'll refactor them later